jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -259,6 +259,7 @@ def _match_models(models, channel, degree, center=None, center_cs='image'):
     cbs.channel = str(channel)
     cbs.band = 'ALL'
     cbs.single = True
+    cbs.weighting = 'EMSM'
```

Comment:
What does this do?

Reply:
This forces the cube building routine to use the EMSM instead of (currently) default MSM weighting scheme.  This is an exponential radial weight function instead of a 1/r that a previous ticket/PR demonstrated achieved better performance.  At some point Jane and I planned to make the default cube building for MRS use EMSM, but in the meantime this will ensure that background matching uses cubes with the fewest artifacts.